proto/v3/message_contents/public_key.proto

```diff
@@ -59,8 +61,15 @@ message VmacFallbackKeyRotation {
 }
 
 // A contact bundle served by the server to a requesting client
-message VmacContactBundle {
+message VmacInstallationPublicKeyBundleV1 {
     VmacAccountLinkedKey identity_key = 1;
-    // An unused one time prekey or fallback key returned by the server
-    VmacDeviceLinkedKey prekey = 2;
+    VmacDeviceLinkedKey fallback_key = 2;
```

Comment:
We've been previously using the account's `fallback_key` and calling it the `prekey` in the proto, so this renames for clarity.

Also opens the door for us to add actual `prekey`s into the message later if we need.